Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -580,3 +580,43 @@ export const getImageUrl = function (url) {
   }
 };
 
+export const getPastDays = function (count) {
+  try {
+    let result = [];
+    for (var i = 0; i < count; i++) {
+      let d = new Date();
```

Comment:
this `d` can be renamed to `currentDate ` and can be placed outside the `for` loop as it is not required to initialise it in every iteration.

Reply:
since we are changing same date so moving it outside will add more lines of code to handle case, also renamig it might not be the optimal solution as we keep on changing the same date which will not be current date.